### PR TITLE
tests: added description on how to run specific tests

### PR DESCRIPTION
**Pull request by kayrus**
/cc @jonboulle @antrik @tixxdz

### CODE REVIEW

**Comment by antrik** [comment]
I would write more explicitly that you can just pass regular `go test` arguments, such as `-run`.

That's not a show-stopper, though... Feel free to ignore ;-)

**Comment by jonboulle** [comment]
I kinda agree with @antrik - we should just clarify it's passing on args to go test

**Comment by tixxdz** [comment]
@kayrus could you please update ?

**Comment by tixxdz** [comment]
@kayrus ping

**Comment by kayrus** [comment]
@jonboulle @tixxdz updated.

**Comment by antrik** [comment]
Thanks, that's better :-)

**Comment by tixxdz** [comment]
lgtm to me too!

File: functional/README.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ Since the tests utilize [`systemd-nspawn`][systemd-nspawn], this needs to be inv
...
+```
+
+The test functions list could be printed using `grep` command:
+
+```sh
+$ cd fleet/functional
+$ grep -r 'func Test' .
+```
+
+[Here][golang-test-flags] you can find detailed description on golang test flags.
```
Comment (jonboulle) [inline functional/README.md]: Channelling @joshix, who hates "here" links:
> You can find detailed description of the available test flags in the [Go documentation](golang-test-flags).

File: functional/README.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ Since the tests utilize [`systemd-nspawn`][systemd-nspawn], this needs to be inv
 
 If the tests are aborted partway through, it's currently possible for them to leave residual state as a result of the `systemd-nspawn` operations. This can be cleaned up using the `clean.sh` script.
 
+### Using go flags in functional tests
+
+`fleet/functional` scripts forwards all arguments directly to `go` binary. This allows you to pass `-run regexp` argument to the `./test` script and run specific test function. Example below shows how to run only `TestSchedule*` test functions:
```
Comment (jonboulle) [inline functional/README.md]: nits:
to **the** go binary
and run specific test function(s)
**The** example below

**Comment by kayrus** [comment]
@jonboulle fixed

File: functional/README.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ Since the tests utilize [`systemd-nspawn`][systemd-nspawn], this needs to be inv
...
+```
+
+The test functions list could be printed using `grep` command:
+
+```sh
+$ cd fleet/functional
+$ grep -r 'func Test' .
+```
+
+You can find detailed description of the available test flags in the [Go documentation][golang-test-flags].
```
Comment (jonboulle) [inline functional/README.md]: Argh, I'm sorry, this is totally my bad (!!), but it should be **a** detailed description.

Comment (kayrus) [inline functional/README.md]: NP, fixed

**Comment by jonboulle** [comment]
Thank you <3

**Pull request merged**